bootstrap/bootstrap.sh

```diff
@@ -188,8 +188,8 @@ if [ $forcedCliLocalPath = "<none>" ]; then
     fi
 fi
 
+runtimesPath="$cliInstallPath/shared/Microsoft.NETCore.App"
 if [ $sharedFxVersion = "<auto>" ]; then
-    runtimesPath="$cliInstallPath/shared/Microsoft.NETCore.App"
```

Comment:
Could you change `cliInstallPath` to the same as the (I think) equivalent powershell `CliLocalPath`?